lms/templates/navigation-edx.html

```diff
@@ -79,9 +79,7 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
         <a href="#" class="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span> &#9662;</a>
         <ul class="dropdown-menu" aria-label="More Options" role="menu">
           <%block name="navigation_dropdown_menu_links" >
-            % if settings.MKTG_URL_LINK_MAP.get('FAQ'):
```

Comment:
@symbolist my code reading skills are shaky -- I believe that this is removing the current "Help" link from the username dropdown? This is a part of the acceptance criteria of this story, so I'm fine with it either here or in a separate PR, but it does need to be removed.